src/pytorch_lightning/loggers/mlflow.py

```diff
@@ -269,8 +276,9 @@ def log_metrics(self, metrics: Mapping[str, float], step: Optional[int] = None)
                     category=RuntimeWarning,
                 )
                 k = new_k
+            metrics_list.append(Metric(key=k, value=v, timestamp=timestamp_ms, step=step or 0))
 
-            self.experiment.log_metric(self.run_id, k, v, timestamp_ms, step)
+        self.experiment.log_batch(run_id=self.run_id, metrics=metrics_list)
```

Comment:
Would you mind verifying in which version of mlflow this log_batch API was introduced?

Reply:
> I was today years old when I learned I can query for text in github release notes.

Yes I could have done this myself :(( I'm sorry I bothered you for this.

In this case I agree we should set the minimum version to 1.0.0 (it is about as old as Lightning). And then we should be good :)

Reply:
No, I'm glad you asked! I should have done it originally.

I'm searching around and the only reference to the mlflow dependency is in `loggers.info`, which I updated.